.github/scripts/nova_prescript.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+PRELUDE=.github/scripts/setup_env.bash
+BUILD_ENV=CONDA_ENV
+#Display System Info
+$PRELUDE; print_system_info
+#Display GPU Info
+$PRELUDE; print_gpu_info
+#Install C/C++ Compilers
+$PRELUDE; install_cxx_compiler $BUILD_ENV
+#Install Build Tools
+$PRELUDE; install_build_tools $BUILD_ENV
+#Install CUDA
+$PRELUDE; install_cuda $BUILD_ENV "$CU_VERSION"
+# Install PyTorch Nightly
+$PRELUDE; install_pytorch_pip $BUILD_ENV nightly cuda "$CU_VERSION"
```

Comment:
CUDA and the right pytorch version should already be installed for you under the hood by our pipeline, there is no need to do so in your pre-script.

Reply:
Thank you so much, I've removed them.